src/components/dialogs/searchDialog/SearchDialog.vue

```diff
@@ -190,8 +190,8 @@ export default class SearchDialog extends HasTask {
       const response = await api.getSearchResults(responseId);
       const isStopped = response.status === "Stopped";
 
+      this.grid.searchItems = this.grid.searchItems.concat(response.results.slice(this.grid.searchItems.length,response.results.length))
```

Comment:
Looks like can be simplified to
```suggestion
      this.grid.searchItems.splice(-1, 0, ...response.results.slice(this.grid.searchItems.length))
```